### DISCUSSION TITLE
registered user - stripchat downloads no longer work. please get this fixed

### DISCUSSION THREAD

**Discussion by guesswho123123** [aclap-dev/video-downloadhelper#1505] [Help and Q&A]
### Your question

up until recently, purchased videos from stripchat could be downloaded with VDH. as of today, the video does not appear correctly in the DL choices. no matter what video is selected, the live stream is downloaded, usually 6-12 seconds. moreover, the pblm is browser independent. i'm registered with edge firefox and chrome. none work anymore.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"d5480161ba814538"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:133.0) Gecko/20100101 Firefox/133.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "C:\\Users\User\dwhelper",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 6,
    "successfull_dl": 1
}

**Comment by Mattmjs** [comment]
Doesn't the site use DRM ? Do you see this icon ? If not then they must doing something else to stop downloads.

![DRM icon](https://github.com/user-attachments/assets/aac6397c-b4d9-48b6-b7d1-48cc7dfab6d7)

**Comment by Mattmjs** [comment]
Since I'm waiting for a response, I had a look. It said 6 seconds but recorded for 5 minutes before I stop it and no DRM icon.
Maybe it's certain streams like do you have to log in to view or do you have to pay to view it, I don't know.

![stripchat](https://github.com/user-attachments/assets/76d0aeea-67cf-498f-999d-9477bb49cfed)

**Comment by guesswho123123** [comment]
it's working today.... crazy!  i'll keep you posted if anything changes

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

@Mattmjs You went to superchatlive, which does work for me as well.  They are livestreams so duration is impossible to determine.  That's why VDH shows 6 seconds.  The chunk size of the livestream is, apparently, 6 seconds.  That value might be different on other sites.  I also had no problem recording the one random thing I chose.  I stopped it after 1 minute & the clip I had played fine.

But the original poster was saying things weren't working right on stripchat, another site entirely.  When I tried to go to the site, it was allegedly not found.  Downdetector reported that the site was indeed down, so I can't test that one.  I imagine it's very much like superchatlive.  But until I can actually see it, that's just a guess.

**Reply by GrampaWildWilly** [reply]
There are 2 clues that it's a livestream in your image, Matt.  Which I point out not so much to tell you something you don't know but to help anybody else who might be reading this.

![#01](https://github.com/user-attachments/assets/729dde98-7108-4953-9962-e33ef1d19881)

One clue is the livestream indicator.  The other is that the claimed duration on the icon is very much less than the duration VDH has been recording.

**Reply by Mattmjs** [reply]
I referred to the last discussion on this site.
#1300 , apparently it is stripchat but a different 
domain. There are a few discussions on this 
site searching the forum but maybe the issue 
is fixed ?
